### ISSUE TITLE
Batch_Norm Support?

### ISSUE THREAD

**Issue by n3011** [kevinlin311tw/caffe-augmentation#4]
@kevinlin311tw Does this repo support batch norm? Getting error while using batch_norm.

**Comment by kevinlin311tw** [comment]
This repo dosen't support batch normalization.
